starport/pkg/protoanalysis/protoanalysis_test.go

```diff
@@ -39,55 +39,253 @@ func TestLiquidity(t *testing.T) {
-				{Name: "QueryLiquidityPoolResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryLiquidityPoolBatchRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryLiquidityPoolBatchResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryLiquidityPoolsRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryLiquidityPoolsResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryParamsRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryParamsResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchSwapMsgsRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchSwapMsgRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchSwapMsgsResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchSwapMsgResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchDepositMsgsRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchDepositMsgRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchDepositMsgsResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchDepositMsgResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchWithdrawMsgsRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchWithdrawMsgRequest", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchWithdrawMsgsResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "QueryPoolBatchWithdrawMsgResponse", Path: "testdata/liquidity/query.proto"},
-				{Name: "MsgCreatePool", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgCreatePoolRequest", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgCreatePoolResponse", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgDepositWithinBatch", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgDepositWithinBatchRequest", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgDepositWithinBatchResponse", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgWithdrawWithinBatch", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgWithdrawWithinBatchRequest", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgWithdrawWithinBatchResponse", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgSwapWithinBatch", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgSwapWithinBatchRequest", Path: "testdata/liquidity/tx.proto"},
-				{Name: "MsgSwapWithinBatchResponse", Path: "testdata/liquidity/tx.proto"},
-				{Name: "BaseReq", Path: "testdata/liquidity/tx.proto"},
-				{Name: "Fee", Path: "testdata/liquidity/tx.proto"},
-				{Name: "PubKey", Path: "testdata/liquidity/tx.proto"},
-				{Name: "Signature", Path: "testdata/liquidity/tx.proto"},
-				{Name: "StdTx", Path: "testdata/liquidity/tx.proto"},
+				{Name: "PoolRecord", Path: "testdata/liquidity/genesis.proto", Fields: map[string]string{
```

Comment:
Not related to this PR but I think we should simplify a lot the test sources because the current one makes the test very hard to maintain.

A proto test directory with only three messages is sufficient to test the methods